js/src/dapps/signaturereg/IdentityIcon/identityIcon.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+
+import { api } from '../parity';
+import styles from './identityIcon.css';
+
+export default class IdentityIcon extends Component {
```

Comment:
Dapps should not rely on the ui/* stuff - we don't build those as a module and would make splitting of the apps from the repo problematic. However, there is probably something to say fro a `dapps-ui` thing that has all the common components. (e.g. for that IdentityIcon the same code is not in Gavcoin, TokenReg & here - that I know of)

So valid concern, just a different issue.